docs/articles/getting-started.md

```diff
@@ -161,9 +162,9 @@ private void button1_Click(object sender, EventArgs e)
 }
 ```
 
-In some cases you can even access a virtual Grid instead of for example [`Key.A`](xref:Colore.Effects.Keyboard.Key#Colore_Effects_Keyboard_Key_A) you can set the Key in the second row from the top and the fifth column from the left to Red:
-[`Chroma.Instance.Keyboard[1, 4] = ColoreColor.Red;`](xref:Colore.IKeyboard#Colore_IKeyboard_Item_System_Int32_System_Int32_)
-The first int is row and the second is column, starting with `0`! There even are Constants that allow you to loop through all keys:
+In some cases you can even access a virtual Grid instead of for example Key.A you can set the Key in the second row from the top and the fifth column from the left to Red:
+`Chroma.Instance.Keyboard[1, 4] = ColoreColor.Red;`
+The first int is row and the second is column. Starting with 0! There even are Constants that allow you to loop through all Keys:
```

Comment:
"starting with 0" is part of the first sentence so the comma should probably be left in place.

"keys" here is the same as the "grid" usage earlier: a regular word so should not be capitalized. On the other hand, "int" here refers to the C# keyword so should be marked as code with backticks: `int`.